packages/vscode-extension/src/codeLensProvider.ts

```diff
@@ -147,6 +147,7 @@ export class CryptoCodeLensProvider implements vscode.CodeLensProvider {
   }
 }
 
+// Only used to detect adaptive cards in project. Codelens for ac is not used.
 export class AdaptiveCardCodeLensProvider implements vscode.CodeLensProvider {
```

Comment:
Since we already have a function to detect Adaptive Card in workspace (pr #10133), do we still need to keep this class?